File: fizzbuzz.rb

$i = 1
$num = 100

until $i > $num  do
	if $i% 3 ==0 && $i% 5 ==0
		puts ("fizzbuzz")
	elsif $i% 3 ==0
		puts ("fizz")
	elsif $i% 5 == 0 
		puts ("buzz")
	else
		puts($i)
	end
   $i +=1;
end

File: fizzbuzz.cpp
#include <iostream>
using namespace std;

int main() {
	
	for(int x = 1; x < 101; x++){
		if (x%3 == 0){
			cout << "fizz";
		}
		if (x%5 == 0){
			cout << "buzz";
		}
		if (x%3 != 0 && x%5 != 0){
			cout << x;
		}
		cout << endl;
	}
}